src/backend/api/Fusion.Resources.Api/Controllers/Person/PersonController.cs

```diff
@@ -78,27 +82,35 @@ public async Task<ActionResult<List<ApiPersonNote>>> GetPersonNotes(string perso
 
                 r.AnyOf(or =>
                 {
+                    or.BeResourceOwner(user.fullDepartment);
                 });
+
+                // Limited access to other resource owners, only return shared notes.
+                // Give access to all resource owners that share the same L2.
+                r.LimitedAccessWhen(or => or.BeResourceOwner(new DepartmentPath(user.fullDepartment).GoToLevel(3), includeParents: true, includeDescendants: true));
```

Comment:
Just to make sure: L2 = 3 ?

Reply:
I changed that after the comment.. will update comment